src/alt.js

```diff
@@ -494,6 +501,7 @@ class Alt {
 
   addActions(name, ActionsClass) {
     this.actions[name] = this.createActions(ActionsClass)
+    return this.actions[name]
```

Comment:
so I can do
```
this.action = props.alt.addActions(actionName, Action);
```

and save a line of code. it makes sense to me and saves a line of code.